src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx

```diff
@@ -41,9 +41,9 @@ import type {
   DownloadResourceButtonNameValueType,
 } from "../../../../../../../../browser-lib/avo/Avo";
 import useAnalyticsPageProps from "../../../../../../../../hooks/useAnalyticsPageProps";
-import LessonOverview from "../../../../../../../../components/LessonOverview/LessonOverview";
 import { ViewType } from "../../../../../../../../common-lib/urls";
 import getPageProps from "../../../../../../../../node-lib/getPageProps";
+import LessonDetails from "../../../../../../../../components/LessonDetails/LessonDetails";
```

Comment:
We need to look into aliases to avoid these sorts of paths

Reply:
Yes, I think @lioneljingles was looking at setting it up.

The NextJS default now is for `@/*` to resolve to `src/*`, which I think it would be a good idea to add

This can be achieved by adding:
```json
    "paths": {
      "@/*": ["./src/*"]
    }
```
to `complierOptions` in tsconfig.json

The next thing would be to look into a codemod to refactor exisitng code to adhere this standard.

Not found one per se, but seen a couple of pointers [here (though they're doing the reverse](https://stackoverflow.com/questions/57188027/refactor-aliased-imports-to-relative-paths) and [here](https://github.com/s-yadav/relative-to-alias)